azurerm/internal/services/applicationinsights/resource_arm_application_insights.go

```diff
@@ -89,6 +89,19 @@ func resourceArmApplicationInsights() *schema.Resource {
 
 			"tags": tags.Schema(),
 
+			"daily_cap": {
```

Comment:
This might be better as 
```suggestion
			"daily_traffic_cap_in_gb": {
```

Reply:
This option is related to data which is actually stored in applications insight. So my recommendation would be **daily_data_cap_in_gb**. What you think?

Reply:
works for me!